### ISSUE TITLE
Make search fields tokenized

### ISSUE THREAD

**Issue by GVogeler** [acdh-oeaw/apis-instance-manmax#13]
currently all search fields in the default APIS interface search for full strings only, i.e. sequence matters and the search term includes whitespace ("Herzog Luwdig von Bayern" != "Ludwig, Herzog von Bayern".) Using PostGres full text capabilities might be useful for searching and for autocomplete? 

See #5 as well.

**Comment by GVogeler** [comment]
Gendering Maximilian: "Die Suche nach Namen im Factoid-Editor ist ja eine full string Suche, also müsste man normieren, wie Labels für Personen vergeben werden, sonst entstehen sicherlich viele Dopplungen."

**Comment by GVogeler** [comment]
addressed for FactoidBuilder in #31. Extend the solution to Vanilla APIS?

**Comment by richardhadden** [comment]
I don't think this is functionality that we can add to the default APIS: at least, changing the default behaviour that considerably would need agreement of APIS team (it's not something that can be easily overridden)
